packages/builders/src/interactions/commands/chatInput/options/attachment.ts

```diff
@@ -5,6 +5,9 @@ import { ApplicationCommandOptionBase } from './ApplicationCommandOptionBase.js'
  * A chat input command attachment option.
  */
 export class ChatInputCommandAttachmentOption extends ApplicationCommandOptionBase {
+	/**
+	 * Creates a new attachment option from API data.
```

Comment:
```suggestion
	 * Creates a new attachment option.
```

Reply:
FYI this is how it's written in most (if not all) the other existing builders. I'll change them too as I also don't think "from API data" belongs in the constructor description, but the `data` parameter itself